### PR TITLE
refactor: Enhancing type for IPoolsInfosProps in the CommunityPoolsTable

### PR DESCRIPTION
**Pull request by ChristanDaniel**
### Why 
 Pull Request with a change request has been accepted, this includes the requested modification.

### Changes
- Enhancing type for IPoolsInfosProps in the CommunityPoolsTable

### CODE REVIEW

**Review by jonyreis — APPROVED** [review]

**Review by ThallesP — APPROVED** [review]
LGTM

File: src/templates/Explore/CommunityPoolsTable/index.tsx [hunk 1]
```diff
@@ -23,53 +23,43 @@ import {
...
+type UnderlyingAssets = {
+  token: {
+    logo?: string | null
+    wraps?: {
+      logo?: string | null
+    } | null
+  }
+}
+
+interface IPoolsInfosProps {
```
Comment (nicholascostadev) [inline src/templates/Explore/CommunityPoolsTable/index.tsx:35]: You changed it to interface for any reason? It's fine for me, just a question

**Review by nicholascostadev — APPROVED** [review]
LGTM

File: src/templates/Explore/CommunityPoolsTable/index.tsx (hunk 1 above)
Comment (ChristanDaniel) [reply src/templates/Explore/CommunityPoolsTable/index.tsx:35]: This is a standard

**Pull request merged**